src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -545,6 +554,73 @@ private ImmutableArray<AdditionalTextFile> ResolveAdditionalFilesFromArguments(L
             return builder.ToImmutableArray();
         }
 
+        private static void ReportAnalyzerExecutionTime(TextWriter consoleOutput, AnalyzerDriver analyzerDriver, CultureInfo culture, bool isConcurrentBuild)
+        {
+            Debug.Assert(analyzerDriver.AnalyzerExecutionTimes != null);
+            if (analyzerDriver.AnalyzerExecutionTimes.IsEmpty)
+            {
+                return;
+            }
+
+            var totalAnalyzerExecutionTime = analyzerDriver.AnalyzerExecutionTimes.Sum(kvp => kvp.Value.TotalSeconds);
```

Comment:
Oh, I see that Enumerable.Sum is not generic. There's no overload for TimeSpan. Oh well, never mind then.